packages/web-frontend/src/containers/DrillDownOverlay/index.js

```diff
@@ -12,7 +12,12 @@ import CircularProgress from 'material-ui/CircularProgress';
 import BackIcon from 'material-ui/svg-icons/hardware/keyboard-arrow-left';
 
 import { TRANS_BLACK, DIALOG_Z_INDEX, WHITE } from '../../styles';
-import { attemptDrillDown, closeDrillDown, goToDrillDownLevel } from '../../actions';
+import {
+  attemptDrillDown,
+  closeDrillDown,
+  goToDrillDownLevel,
+  setDrillDownDateRange,
```

Comment:
The new import is not used - but this could be a good alternative! I like the idea of this component dealing with as much of the drill down specific stuff as possible

Reply:
Yes good idea. I moved the `onSetDate` handling to the drillDown component and it makes much more sense